tests/knownIssues.js

```diff
@@ -4,7 +4,9 @@ module.exports = [
 	// chapling brunch does not mark items as completed via the completed CSS class
 	'TodoMVC - chaplin-brunch, No Todos, should hide #main and #footer',
 	'TodoMVC - chaplin-brunch, Mark all as completed, should allow me to mark all items as completed',
+	/* jscs:disable */
```

Comment:
Is there a reason you're using multiline-`/*` comments here while using `//` for the others?